src/main/java/org/jabref/gui/search/SearchResultsTable.java

```diff
@@ -0,0 +1,64 @@
+package org.jabref.gui.search;
+
+import javax.swing.undo.UndoManager;
+
+import javafx.scene.control.SelectionMode;
+import javafx.scene.control.TableView;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.externalfiletype.ExternalFileTypes;
+import org.jabref.gui.maintable.BibEntryTableViewModel;
+import org.jabref.gui.maintable.MainTable;
+import org.jabref.gui.maintable.MainTableColumnFactory;
+import org.jabref.gui.maintable.MainTableDataModel;
+import org.jabref.gui.maintable.MainTablePreferences;
+import org.jabref.gui.maintable.SmartConstrainedResizePolicy;
+import org.jabref.gui.maintable.columns.MainTableColumn;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.preferences.PreferencesService;
+
+public class SearchResultsTable extends TableView<BibEntryTableViewModel> {
```

Comment:
As this class has quite a bit of code duplication with the main table, can we have one class that handles the display of entries (in a table). Would make it easier to reuse it.

Reply:
I know, but it's just part of the constructor and this is exactly what the want. Show the user a table he's familiar with

Reply:
I think this would escalate this PR way too much. Probably something for a later code quality PR.